### PR TITLE
Update the issute template for new tool

### PR DESCRIPTION
**Pull request by ybiquitous**
This aims to make it easier writing an issue for a new tool by doing:

- Reduce questions
- Arrange formats
- Clarify descriptions
- Remove the links to other implemented tools and the To-do section.
  - Because I think we should write the developer guide for them. (I'll do it later.)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-tool-support-request.md [hunk 1]
```diff
@@ -4,45 +4,22 @@ about: Suggest new analysis tool support for this project
...
-- Official Site/Documentation: [URL]
+## Why do you need the tool?
 
-### Supplementary information
+<!-- (Required) Input your motivation or reason to support the tool. -->
 
-This issue is similar to these completed issues.
+## Additional context
 
-- cpplint support
```
Comment (sumyapp) [inline .github/ISSUE_TEMPLATE/new-tool-support-request.md]: I think, if removed this information in this template, contributors can not understand to do these issues.
So can you create CONTRIBUTING.md file or update README.md file?

Comment (ybiquitous) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: > So can you create CONTRIBUTING.md file or update README.md file?

Yes, I will create `CONTRIBUTING.md`!

Comment (ybiquitous) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: Add the contributing guide via #711.

Comment (ybiquitous) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: I will add a new pull request template for alternative to this To-do section.

Comment (ybiquitous) [reply .github/ISSUE_TEMPLATE/new-tool-support-request.md]: Added via #713.

**Pull request merged**